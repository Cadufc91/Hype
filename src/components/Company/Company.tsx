import * as React from 'react';
import {
    CompanyContainer,
    CompanyText,
    ColoredText,
    LogoContainer,
    LogoItem,
    LogoImg,
} from './style';
import Youtube from '../../assets/img/youtube.png';
import Google from '../../assets/img/google.png';
import Adobe from '../../assets/img/adobe.png';
import Sketch from '../../assets/img/sketch.png';
import { BsArrowRight } from 'react-icons/bs';

export function Company () {
    return(
        <CompanyContainer>
            <CompanyText>Trabalhei com <ColoredText>289+</ColoredText> empresas ao redor do mundo.</CompanyText>
            <LogoContainer>
                <LogoItem>
                    <LogoImg src={Youtube} alt="" />
                </LogoItem>
                <LogoItem>
                    <LogoImg src={Google} alt="" />
                </LogoItem>
                <LogoItem>
                    <LogoImg src={Adobe} alt="" />
                </LogoItem>
                <LogoItem>
                    <LogoImg src={Sketch} alt="" />
                </LogoItem>
                <LogoItem>
                    <BsArrowRight size={32}/>
                </LogoItem>
            </LogoContainer>
        </CompanyContainer>
    )
};

export default Company;