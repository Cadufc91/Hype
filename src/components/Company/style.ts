import styled from "styled-components";

export const CompanyContainer = styled.section`
    display: flex;
    align-items: center;
    justify-content: center;
    max-width: 68%;
    width: 70%;
    margin-left: 15%;
`;

export const CompanyText = styled.p`
    max-width: 16.25%;
    margin-right: 10px;
    color: #fff;
    font-size: 25px;
    font-weight: 400;
`;

export const ColoredText = styled.span`
    color: #ED1CA6;
`;

export const LogoContainer = styled.div`
    display: flex;
    align-items: center;
    justify-content: center;
    height: 120px;
`;

export const LogoItem = styled.div`
    margin-right: 20px;
    height: 125px;
    width: 125px;
    background: #fff;
    display: flex;
    align-items: center;
    justify-content: center;
`;
    
export const LogoImg = styled.img`
    width: 70%;
`;