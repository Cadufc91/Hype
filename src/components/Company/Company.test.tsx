import { render } from "@testing-library/react";
import  { Company } from './Company';

describe('Header', () => {
    it('should match snapshot', () => {
        const { container } = render(<Company />)

        expect(container.firstChild).toMatchSnapshot()
    })
})