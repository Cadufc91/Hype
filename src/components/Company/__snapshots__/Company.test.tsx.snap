// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<section
  class="sc-bczRLJ iQAcfI"
>
  <p
    class="sc-gsnTZi kSuNDB"
  >
    Trabalhei com 
    <span
      class="sc-dkzDqf bEnnWw"
    >
      289+
    </span>
     empresas ao redor do mundo.
  </p>
  <div
    class="sc-hKMtZM kNfDcb"
  >
    <div
      class="sc-eCYdqJ efgUtu"
    >
      <img
        alt=""
        class="sc-jSMfEi fNgAYy"
        src="youtube.png"
      />
    </div>
    <div
      class="sc-eCYdqJ efgUtu"
    >
      <img
        alt=""
        class="sc-jSMfEi fNgAYy"
        src="google.png"
      />
    </div>
    <div
      class="sc-eCYdqJ efgUtu"
    >
      <img
        alt=""
        class="sc-jSMfEi fNgAYy"
        src="adobe.png"
      />
    </div>
    <div
      class="sc-eCYdqJ efgUtu"
    >
      <img
        alt=""
        class="sc-jSMfEi fNgAYy"
        src="sketch.png"
      />
    </div>
    <div
      class="sc-eCYdqJ efgUtu"
    >
      <svg
        fill="currentColor"
        height="32"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 16 16"
        width="32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8z"
          fill-rule="evenodd"
        />
      </svg>
    </div>
  </div>
</section>
`;
